cypress/page/student-registration.page.ts

```diff
@@ -0,0 +1,134 @@
+import * as dayjs from 'dayjs'
+class StudentRegistrationPage {
+    private studentRegistrationPageURL: string;
+    private firstNameInput: string;
+    private lastNameInput: string;
+    private emailInput: string;
+    private genderComponent: string;
+    private mobileInput: string;
+    private dateOfBirthInput: string;
+    private hobbiesComponent: string;
+    private currentAddressInput: string;
+    private stateInput: string;
+    private cityInput: string;
+    private submitButton: string;
+    private Modaltitle: string;
+    private valuesAfterSave: string;
+
+    constructor() {
+        this.studentRegistrationPageURL = "https://demoqa.com/automation-practice-form"
+        this.firstNameInput = "#firstName";
+        this.lastNameInput = "#lastName";
+        this.emailInput = "#userEmail";
+        this.genderComponent = "#genterWrapper";
+        this.mobileInput = "#userNumber";
+        this.dateOfBirthInput = "#dateOfBirthInput";
+        this.hobbiesComponent = "#hobbiesWrapper input+label";
+        this.currentAddressInput = "#currentAddress";
+        this.stateInput = "#state .css-1wy0on6";
+        this.cityInput = "#city .css-1wy0on6";
+        this.submitButton = "#submit";
+        this.Modaltitle = "#example-modal-sizes-title-lg";
+        this.valuesAfterSave = "tr td";
+    }
+
+    public goToStudentRegistrationForm(): void {
+        cy.visit(this.studentRegistrationPageURL);
+    }
+
+    public fillInForm(personalInformation: any = {}): void {
```

Comment:
instead of defining any type, could you define an interface or type? see reference: https://www.typescriptlang.org/docs/handbook/2/objects.html

Reply:
Resolved!